frontend/packages/console-shared/src/components/formik-fields/multi-column-field/MultiColumnFieldRow.tsx

```diff
@@ -32,14 +34,24 @@ const MultiColumnFieldRow: React.FC<MultiColumnFieldRowProps> = ({
   isReadOnly,
   disableDeleteRow,
   spans,
+  complexFields = [],
```

Comment:
Why do we need to have an array and store value for every column? We could simply have this flag at the row level. WDYT?